pkg/rule/testrule.go

```diff
@@ -12,6 +12,28 @@ var TestRule = Rule{
 	},
 }
 
+var TestErrorRule = Rule{
+	Name:         "slave",           // wokeignore:rule=slave
+	Terms:        []string{"slave"}, // wokeignore:rule=slave
+	Alternatives: []string{"follower"},
+	Severity:     0,
+	Options: Options{
+		WordBoundary: false,
+	},
+}
+
+var TestInfoRule = Rule{
+	Name:         "test",           // wokeignore:rule=slave
+	Terms:        []string{"test"}, // wokeignore:rule=slave
```

Comment:
These don't need wokeignore annotations do they?

Reply:
Running `woke` on the codebase returns results for it without. Alternatively I could add it to the project .wokeignore if you prefer?

Without the wokeignore annotations:
```
❯ ./woke
pkg/rule/testrule.go:6:16-25: `whitelist` may be insensitive, use `allowlist`, `inclusion list` instead (warning)
	Name:         "whitelist",
	               ^
pkg/rule/testrule.go:7:25-34: `whitelist` may be insensitive, use `allowlist`, `inclusion list` instead (warning)
	Terms:        []string{"whitelist", "white-list", "whitelisted", "white-listed"},
	                        ^
pkg/rule/testrule.go:7:38-48: `white-list` may be insensitive, use `allowlist`, `inclusion list` instead (warning)
	Terms:        []string{"whitelist", "white-list", "whitelisted", "white-listed"},
	                                     ^
pkg/rule/testrule.go:7:52-61: `whitelist` may be insensitive, use `allowlist`, `inclusion list` instead (warning)
	Terms:        []string{"whitelist", "white-list", "whitelisted", "white-listed"},
	                                                   ^
pkg/rule/testrule.go:7:67-77: `white-list` may be insensitive, use `allowlist`, `inclusion list` instead (warning)
	Terms:        []string{"whitelist", "white-list", "whitelisted", "white-listed"},
	                                                                  ^
pkg/rule/testrule.go:16:16-21: `slave` may be insensitive, use `follower`, `replica`, `standby` instead (error)
	Name:         "slave",
	               ^
pkg/rule/testrule.go:17:25-30: `slave` may be insensitive, use `follower`, `replica`, `standby` instead (error)
	Terms:        []string{"slave"},
```

Reply:
You need them in L16 and L17, but not in L26 and L27 because `slave` is not in this line

Reply:
Umm, yeah. I don't really have an excuse for that one :) Sorry about that. Fixed.